QEfficient/exporter/export_hf_to_cloud_ai_100.py

```diff
@@ -43,49 +40,26 @@ def convert_to_cloud_bertstyle(
         tokenizer (HF AutoTokenizer): Tokenzier to prepare inputs.
         model_path (str, optional): The path where the model is stored. If None, the model is loaded from the default location.
         seq_len (int, optional): The length of the sequence. Default is 128.
-        return_path (bool): If True, return the base path for models and exported onnx model path
-        save_fp32_onnx (bool); If True, fp32 unclipped version of ONNX will be saved. Default is False.
-        save_fp16_onnx (bool); If false, generation of fp32 clipped version of ONNX will be skipped. Default is True.
 
     """
     if os.path.exists(onnx_dir_path):
         logger.warning(f"Overriding {onnx_dir_path}")
         shutil.rmtree(onnx_dir_path)
-
-    if not (save_fp32_onnx or save_fp16_onnx):
-        raise AttributeError("save_fp32_onnx and save_fp16_onnx can't be false")
-
-    if tokenizer.padding_side != "left":
-        logger.warning("Please use padding_side='left' while initializing the tokenizer")
-        tokenizer.padding_side = "left"
-
-    if tokenizer.pad_token_id is None:
-        tokenizer.pad_token_id = tokenizer.eos_token_id
-
+        
     # Decide path for saving exported ONNX files.
-    fp32_model_name, fp16_model_name = export_bertstyle_model_to_onnx(
-        model_name, qeff_model.model, tokenizer, onnx_dir_path, seq_len, save_fp32_onnx, save_fp16_onnx
-    )  # type: ignore
-
+    model_name = export_bertstyle_model_to_onnx(model_name, qeff_model.model, tokenizer, onnx_dir_path, seq_len) # type: ignore
+  
     # return the model path for automation.
-    if return_path:
-        if save_fp16_onnx:
-            return onnx_dir_path, os.path.join(onnx_dir_path, f"{fp16_model_name}.onnx")
-        else:
-            return onnx_dir_path, os.path.join(onnx_dir_path, f"{fp32_model_name}.onnx")
+    return onnx_dir_path, os.path.join(onnx_dir_path, f"{model_name}.onnx")
```

Comment:
return single path? no need to return onnx_dir_path

Reply:
Then this requires changes across this `Api`() usage, check top level api and notebooks, if they need this change.

Reply:
CI checks are failing